src/mito2/src/region.rs

```diff
@@ -292,7 +295,9 @@ impl MitoRegion {
         let manifest_usage = self.stats.total_manifest_size();
         let num_rows = version.ssts.num_rows() + version.memtables.num_rows();
         let manifest_version = self.stats.manifest_version();
-        let flushed_entry_id = version.flushed_entry_id;
+        let flushed_entry_id = version
+            .flushed_entry_id
+            .max(self.high_watermark_since_flush.load(Ordering::Relaxed));
```

Comment:
I think they are two different things. Please add a new field.

Reply:
Renamed to `topic_latest_entry_id` and added a new field in [RegionStatistics] for it.